operator/roles/kiali-deploy/tasks/openshift/os-main.yml

```diff
@@ -54,3 +54,8 @@
   include: openshift/os-oauth.yml
   when:
   - is_openshift == True
+
```

Comment:
above this "Process Kiali OAuth client on OpenShift" task is where we should now do the include of the yaml that will create the kiali_route_url fact.

```
- name: Get the Kiali Route URL
  include: openshift/os-get-kiali-route-url.yml
  when:
￼  - is_openshift == True
```